api_tester/lib/api_tests/purchases_flutter_api_test.dart

```diff
@@ -494,4 +501,9 @@ class _PurchasesFlutterApiTest {
     Future<void> future = Purchases.syncObserverModeAmazonPurchase(
         productID, receiptID, amazonUserID, isoCurrencyCode, price);
   }
+
+  void _showInAppMessages() async {
+    Future<void> future = Purchases.showInAppMessages(types: {InAppMessageType.billingIssue,
+      InAppMessageType.priceIncreaseConsent, InAppMessageType.generic});
```

Comment:
We should add one more API test for this enum, to make sure we validate all possible values of the enum